babyagi.py

```diff
@@ -145,6 +145,24 @@ def context_agent(query: str, n: int):
     sorted_results = sorted(results.matches, key=lambda x: x.score, reverse=True)
     return [(str(item.metadata['task'])) for item in sorted_results]
 
+# OannesGPT Additions - Function to write result to file and append on top of existing file  
+def write_result_to_file(filename, result):
+    # Try to open the output file for reading
+    try:
+        with open(filename, 'r') as f:
+            existing_data = f.read()
+    except FileNotFoundError:
+        existing_data = ''
+
+    # Combine the existing data and the new variable
+    new_data = result + '\n\n' + existing_data
+
+    # Write the new data to the output file
+    with open(filename, 'w') as f:
+        f.write(new_data)
+
+# End of: Oannes Additions
```

Comment:
delete

Reply:
@davidbyttow Hey, I updated the file without notes ;)Sorry am new to this, wanted to be clear